src/app/page.scss

```diff
@@ -1 +1,3 @@
-@import "styles/pages/home.scss";
\ No newline at end of file
+@import "styles/pages/home.scss";
+@import "styles/pages/about.scss";
```

Comment:
We might want to remove the about and pitch css imports from page.scss. We can reorganize if/when we move pages to the `pages/` folder.